metriq_gym/benchmarks/quantum_volume.py

```diff
@@ -219,8 +234,14 @@ def poll_handler(
             raise TypeError(f"Expected job_data to be of type {type(QuantumVolumeData)}")
 
         stats: AggregateStats = calc_stats(job_data, flatten_counts(result_data))
+
         return QuantumVolumeResult(
             num_qubits=job_data.num_qubits,
             confidence_pass=stats.confidence_pass,
-            xeb=1,
+            xeb=stats.avg_xeb,
+            hog_prob=stats.hog_prob,
+            hog_pass=stats.hog_pass,
+            eplg=stats.avg_eplg,
+            p_value=stats.p_value,
+            trials=stats.trials,
```

Comment:
Do we want the full `trials` object in the final output? Let's think in terms of eventually uploading this to metriq.info.